src/Files.Backend/Data/Enums/SearchBoxTextChangeReason.cs

```diff
@@ -3,12 +3,24 @@
 
 namespace Files.Backend.Data.Enums
 {
+	/// <summary>
+	/// Defines constants that specify search box text changed reason.
+	/// </summary>
 	public enum SearchBoxTextChangeReason : ushort
 	{
+		/// <summary>
+		/// The user has changed text.
+		/// </summary>
 		UserInput,
 
+		/// <summary>
+		/// The Files has programatically changed.
```

Comment:
*files

Reply:
I meant the Files app. Should I change to The app has…?

Reply:
What does it mean by "changed"

Reply:
changed text in SearchBox.

Reply:
How about "Search box text has changed"?

Reply:
Yeah, there’s some reasons why it is changed. UserInput (the user has changed it by their keyboard or sth, Programmatic (The app has changed it, like autofill or sth), SuggestChosen(the user has chosen a suggest of AutoSuggestTextBox.